teams/teams-ps/teams/Remove-CsTeamsShiftsPolicy.md

```diff
@@ -23,25 +23,26 @@ Remove-CsTeamsShiftsPolicy [-Identity] <XdsIdentity> [<CommonParameters>]
 
 ### Example 1
 ```powershell
-PS C:\> Remove-CsTeamsShiftsPolicy -Identity IsaiahL
+PS C:\> Remove-CsTeamsShiftsPolicy -Identity OffShiftAccess_WarningMessage1_AlwaysShowMessage
 ```
 
-In this example, the TeamsShiftsPolicy assigned to the user "IsaiahL" is removed.
+In this example, the policy instance to be removed is called "OffShiftAccess_WarningMessage1_AlwaysShowMessage". 
+
 
 ## PARAMETERS
 
 ### -Identity
-UserId or user's email id.
+Policy instance name.
 
 ```yaml
 Type: XdsIdentity
-Parameter Sets: (All)
+Parameter Sets: Identity

```

Comment:
```suggestion
Parameter Sets: (All)
```

Reply:
I've checked with PlatyPs and there is no identity parameter set so we use (All)